weblogic-azure-aks/src/main/arm/scripts/buildWLSDockerImage.sh

```diff
@@ -117,6 +117,29 @@ function initialize() {
     mkdir wlsdeploy/domainLibraries
 }
 
+function download_wdt_wit() {
+    local wlsToolingFamilyJsonFile=weblogic_tooling_family.json
+    # download the json file that has wls operator version from weblogic-azure repo.
+    curl -m ${curlMaxTime} -fsL "${gitUrl4WLSToolingFamilyJsonFile}" -o ${wlsToolingFamilyJsonFile}
+    if [ $? -eq 0 ]; then
```

Comment:
I would like to understand why you are not using the `set -Eeuo pipefail` best practice in this and other scripts?